### ISSUE TITLE
How to make the output of `fd` use the same colors as the readme example

### ISSUE THREAD

**Issue by erusev** [sharkdp/fd#375]
How to make the output of `fd` use the same colors as the readme example:

![image](https://user-images.githubusercontent.com/184170/50483824-1aef3580-09f6-11e9-8ee8-41601a183886.png)

Is it a separate library that I can install?

Thanks!

**Comment by sharkdp** [comment]
Colors are controlled by the `LS_COLORS` environment variable (see README: https://github.com/sharkdp/fd#colorized-output).

The colors in the screenshot are from a previous custom `LS_COLORS` theme of mine. I am now using [vivid](https://github.com/sharkdp/vivid) to generate my `LS_COLORS` environment variable. It is a new project of mine which allows you to choose between different color themes. Maybe this could be of use to you. If you want the exact same colors as in the screenshot, I would have to see if I can still find that old theme.

**Comment by erusev** [comment]
Awesome!

> If you want the exact same colors as in the screenshot, I would have to see if I can still find that old theme.

No need, I should be fine with the latest version :)

Thanks so much!

**Comment by sharkdp** [comment]
Great :+1: 

Let me know if you have trouble setting up `vivid` (preferably on the issue tracker for `vivid`).

**Comment by condemil** [comment]
It would be great if you can find that old theme. I checked the themes in vivid, but the colors is too different.

**Comment by sharkdp** [comment]
Which vivid-theme did you try? "molokai" should be pretty close.

**Comment by condemil** [comment]
I tried molokai, but the colors is way different:

<img width="425" alt="Screenshot 2019-07-26 at 00 44 20" src="https://user-images.githubusercontent.com/722990/61913831-9f7b4580-af3e-11e9-8ea5-0326347d78f4.png">

**Comment by condemil** [comment]
TL;DR It is Atom One Dark Theme

🕵: The demo file is actually a svg. Based on [commit history](https://github.com/sharkdp/fd/commit/6d83eea32bfe3b052ff20f5cc36f3aacf8067ce0) and [screencast.sh](https://github.com/sharkdp/fd/blob/master/doc/screencast.sh) the svg was generated with [svg-term](https://github.com/marionebl/svg-term). Colors from [screencast.svg](https://github.com/sharkdp/fd/blob/6d83eea32bfe3b052ff20f5cc36f3aacf8067ce0/doc/screencast.svg) file matches with svg-term's [default-theme.ts](https://github.com/marionebl/svg-term/blob/master/src/default-theme.ts) once converted from hex to int. In the [README.md](https://github.com/marionebl/svg-term/blob/master/README.md) it is mentioned that the default theme is Atom One.
